app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>RailsBlogApp</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
```

Comment:
- [ ] When I run the app I got an error, please kindly fix that 🙏🏾 

<img width="566" alt="Screenshot 2023-05-13 at 16 20 18" src="https://github.com/Mukumbuta/My_Bloging_App/assets/74501523/12a60614-38c3-46c3-8bf5-6f860646af40">

- [ ] Also kindly make sure your design match to the given wireframe, which means the margin, padding, borders,....

The given design for every page

<img width="452" alt="Screenshot 2023-05-13 at 16 25 29" src="https://github.com/Mukumbuta/My_Bloging_App/assets/74501523/616af0d9-0ec3-4f31-9f3b-5533ec625f89">